doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,119 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
+manipulates a 'trackball'. In its simplest form (``mouserotationstyle: trackball``),
+the trackball rotates around an in-plane axis perpendicular to the mouse motion
+(it is as if there is a plate laying on the trackball; the plate itself is fixed
+in orientation, but you can drag the plate with the mouse, thus rotating the ball).
+This is more natural to work with than the ``azel`` style; however,
+the plot cannot be easily rotated around the viewing direction - one has to
+drag the mouse in circles with a handedness opposite to the desired rotation.
+
+A different variety of trackball rotates along the shortest arc on the virtual
+sphere (``mouserotationstyle: arcball``); it is a variation on Ken Shoemake's
+ARCBALL [Shoemake1992]_. Rotating around the viewing direction is straightforward
+with it. Shoemake's original arcball is also available
+(``mouserotationstyle: Shoemake``); it is free of hysteresis, i.e.,
+returning mouse to the original position returns the figure to its original
+orientation, the rotation is independent of the details of the path the mouse
+took. However, Shoemake's arcball rotates at twice the angular rate of the
+mouse movement (it is quite noticeable, especially when adjusting roll).
+So it is a trade-off.
+
+Shoemake's arcball has an abrupt edge; this is remedied in Holroyd's arcball
+(``mouserotationstyle: Holroyd``).
+
+Henriksen et al. [Henriksen2002]_ provide an overview.
+
+In summary:
+
+.. list-table::
+   :width: 100%
+   :widths: 30 20 20 20 35
+
+   * - Style
+     - traditional [1]_
+     - incl. roll [2]_
+     - uniform [3]_
+     - path independent [4]_
+   * - azel
+     - ✔️
+     - ❌
+     - ❌
+     - ✔️
+   * - trackball
+     - ❌
+     - ～
+     - ✔️
+     - ❌
+   * - arcball
+     - ❌
+     - ✔️
+     - ✔️
+     - ❌
+   * - Shoemake
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+   * - Holroyd
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+
+
+.. [1] The way it was historically; this is also MATLAB's style
+.. [2] Mouse controls roll too (not only azimuth and elevation)
+.. [3] Figure reacts the same way to mouse movements, regardless of orientation (no difference between 'poles' and 'equator')
+.. [4] Returning mouse to original position returns figure to original orientation (no hysteresis: rotation is independent of the details of the path the mouse took)
+
+Try it out by adding a file ``matplotlibrc`` to folder ``matplotlib\galleries\examples\mplot3d``,
+with contents::
+
+    axes3d.mouserotationstyle: arcball
+
+(or any of the other styles), and run a suitable example, e.g.::
+
+    python surfaced3d.py
+
+(If eternal compatibility with the horrors of the past is less of a consideration
+for you, then it is likely that you would want to go with ``arcball``, ``Shoemake``,
+or ``Holroyd``.)
+
+The size of the trackball or arcball can be adjusted by setting
+``rcParams.axes3d.trackballsize``, in units of the Axes bounding box;
```

Comment:
Could you explain how the user should expect behavior to change? I don't think "size of the trackball" is immediately clear.

Reply:
Expanded on it a bit; maybe it needs a picture? (There is a picture in my mind when I think about it.)

Reply:
I came up with:
"The size of the virtual trackball or arcball can be adjusted as well, by setting ``rcParams.axes3d.trackballsize``. This specifies how much mouse motion is needed to obtain a given rotation angle (when near the center), and it controls where the edge of the arcball is (how far from the center, how close to the plot edge).
The size is specified in units of the Axes bounding box, i.e., to make the trackball span the whole bounding box, set it to 1.
A size of about 2/3 appears to work reasonably well; this is the default."